tests/raftstore/cluster.rs

```diff
@@ -163,10 +167,11 @@ impl<T: Simulator> Cluster<T> {
     }
 
     pub fn call_command(&self,
+                        node_id: u64,
```

Comment:
In my opinion, it's better to have a `node_id` explicitly. In tests we should simulate reality as much as we can, while in the real implementation, where the command is sent has nothing to do with the content inside the command.